src/pages/programs/north-county-and-expansion.js

```diff
@@ -187,7 +187,8 @@ const NorthCountyAndExpansion = ({ componentName }) => {
 					programDescription={`Students will learn music fundamentals and music production skills including navigating a Digital Audio Workstation (DAW), sampling, creating percussive loops, melody composition, and mixing.`}
 					programFootText='Students will then use their music technology knowledge and skills to create their own song! Some of the technologies that will be covered will be: SoundTrap, Novation LaunchPad, Playground Sessions and more.'
 					btnColor='purple'
-					href='https://www.hisawyer.com/treobytes/schedules/activity-set/673279'
+					// intending to remove the buttons on this page
```

Comment:
these commented lines (190-191, 210-211, 230-231, 250-251) can be removed. It's explanatory in the ProgramContainer component